pyiron/sphinx/base.py

```diff
@@ -1399,13 +1432,13 @@ def validate_ready_to_run(self):
         }
 
         if np.any([len(all_groups[group]) == 0 for group in all_groups]):
-            # warnings.warn("The following input groups have not been loaded:\n"
-            #     + ", ".join([
-            #         g for g in all_groups if len(all_groups[g]) == 0
-            #         ])
-            #     + "\n"
-            #     + "They are set to default values."
-            # )
+            warnings.warn("The following input groups have not been loaded:\n"
```

Comment:
This reverts - https://github.com/pyiron/pyiron/pull/684 - from my perspective a default job should not raise a warning. In particular when we present pyiron to new users warnings always look like something is wrong/ unstable.

Reply:
Yeah sorry I didn't mean to uncomment it. Somehow I missed some commits locally when I git pulled.